### ISSUE TITLE
10 - Design Lab

### ISSUE THREAD

**Issue by lespavl** [lespavl/beetroot-academy#7]
Сверстать макет “Design Lab”. [Шаблон](http://academy.beetroot.se/wp-content/uploads/2015/10/design-lab.zip)

Требования:

* Первый блок должен быть размером на всю высоту экрана
* Topworkz, Joint, Worktop и т.д. блоки должны быть ссылками с изменением прозрачности при наведении.
* В мобильной версии текст должен быть на весь экран, а ссылки должны выстроиться по 2 или по 1 в ряд (посмотрите, как будет симпатичнее)

**Comment by Meolie** [comment]
http://meolie.github.io/design-lab/
Только немного съезжают блоки, перекрывающие border-ы.

**Comment by EvgeniiaMoi** [comment]
http://evgeniiamoi.github.io/Design-lab/

**Comment by pro100264** [comment]
https://github.com/pro100264/design_lab.git

**Comment by 3109** [comment]
https://github.com/3109/HW9Designlab.git

**Comment by mrKru** [comment]
http://mrkru.github.io/design_lab
https://github.com/mrKru/design_lab/tree/gh-pages

**Comment by olgaste** [comment]
http://olgaste.github.io/design-lab
https://github.com/olgaste/design-lab.git
Леся, у меня блоки с прерывающимся бордером реализованы 2-мя способами. Один из них (с белым бэкграундом) - простой, но немного странный)) Так можно вообще делать?))

**Comment by lespavl** [comment]
да, так можно делать, почему бы и нет) Главное, чтоб под ним не оказался цветной бекграунд
